lib/rubocop/config_loader.rb

```diff
@@ -35,7 +35,6 @@ def clear_options
       end
 
       def load_file(file)
-        return if file.nil?
```

Comment:
I removed this line to squeeze under the `Metrics/AbcSize` limit. All tests still pass. I figure this an old special case that's not relevant anymore.